Connection.php

```diff
@@ -647,8 +653,20 @@ public function executeCommand($name, $params = [])
         }
 
         \Yii::trace("Executing Redis Command: {$name}", __METHOD__);
+        if ($this->retries > 0) {
```

Comment:
``` php
$tries = $this->retries + 1;
while ($tries--) {
    try {
    } catch () {
    }
}
```

This might be better, and also removed the duplicated codes (`fwrite` and `parseResp`)

Reply:
looks better code wise, but it has the problem, that even if retry is disabled, the connection is reset on the first error.